nuxeo-drive-core/src/test/java/org/nuxeo/drive/service/TestNuxeoDriveManager.java

```diff
@@ -664,6 +665,9 @@ protected boolean isUserSubscribed(String userName, DocumentModel container) {
 
     protected void setPermissions(DocumentModel doc, ACE... aces) {
         ACP acp = session.getACP(doc.getRef());
+        if (acp == null) {
+            acp = new ACPImpl();
+        }
```

Comment:
Or `ACP acp = Optional.ofNullable(session.getACP(doc.getRef())).orElse(new ACPImpl());`

Reply:
`orElseGet(ACPImpl::new)` is better to avoid unnecessary instantiation.

Reply:
That's hardly more readable than a very simple `if` though...